src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/PerformanceFormHandler.php

```diff
@@ -27,15 +27,14 @@
 namespace PrestaShopBundle\Form\Admin\AdvancedParameters\Performance;
 
 use PrestaShop\PrestaShop\Adapter\Feature\CombinationFeature;
-use PrestaShop\PrestaShop\Core\Form\AbstractFormHandler;
 use PrestaShop\PrestaShop\Core\Form\FormDataProviderInterface;
 use Symfony\Component\Form\FormFactoryInterface;
 
 /**
  * This class manages the data manipulated using forms
  * in "Configure > Advanced Parameters > Performance" page.
  */
-final class PerformanceFormHandler extends AbstractFormHandler
+final class PerformanceFormHandler
```

Comment:
I needed to remove this class (which does not exist anymore) else the autowiring would fail (see my PR description)

Reply:
If I understand this class is actually not used any more? So you just clean it to avoid a bug during container compilation?
The weirdest thing is that I can't fin any version where `AbstractFormHandler` actually exists.. it was introduced during 1.7.4 development, but it seems like it was removed before the release but I can't find when

Reply:
Correct. Useless class but I dont know what SemVer says about useless classes so I prefer to let it here.